easybuild/easyconfigs/l/LibSoup/LibSoup-2.66.1-GCCcore-8.2.0.eb

```diff
@@ -0,0 +1,45 @@
+easyblock = 'MesonNinja'
+
+name = 'LibSoup'
+version = '2.66.1'
+
+homepage = 'https://wiki.gnome.org/Projects/libsoup'
+description = """libsoup is an HTTP client/server library for GNOME. It
+uses GObjects and the glib main loop, to integrate well with GNOME
+applications, and also has a synchronous API, for use in threaded
+applications."""
+
+toolchain = {'name': 'GCCcore', 'version': '8.2.0'}
+toolchainopts = {'cstd': 'gnu89'}
+
+source_urls = [FTPGNOME_SOURCE]
+sources = [SOURCELOWER_TAR_XZ]
+checksums = ['4a2cb6c1174540af13661636035992c2b179dfcb39f4d3fa7bee3c7e355c43ff']
+
+builddependencies = [
+    ('binutils', '2.31.1'),
+    ('Meson', '0.50.0', '-Python-3.7.2'),
+    ('Ninja', '1.9.0'),
+    ('GObject-Introspection', '1.60.1', '-Python-3.7.2'),
+]
+
+dependencies = [
+    ('libxml2', '2.9.8'),
+    ('SQLite', '3.27.2'),
+    ('GLib', '2.60.1'),
+    ('libpsl', '0.21.0'),
+    ('cURL', '7.63.0'),
+]
+
+configopts = '--libdir lib -Dgssapi=false -Dvapi=false '
+
+modextrapaths = {
+    'XDG_DATA_DIRS': 'share',
+}
+
+sanity_check_paths = {
+    'files': [],
+    'dirs': ['include', 'lib']
```

Comment:
@akesandgren We should make this a bit stricter imho...

```python
sanity_check_paths = {
    'files': ['lib/libsoup-2.4.%s' % SHLIB_EXT, 'lib/libsoup-gnome-2.4.%s' % SHLIB_EXT],
    'dirs': ['include/libsoup-2.4/libsoup', 'include/libsoup-gnome-2.4/libsoup', 'lib/pkgconfig'],
}

Reply:
Ah yes.... I forgot that. I just put some place holders there while working on it, and then completely forgot.